.genignore

```diff
@@ -1,2 +1 @@
-.cargo-ok
```

Comment:
I don't remember what this file does. And why are you removing this?

Reply:
`cargo-generate` ignore these files by default (https://github.com/cargo-generate/cargo-generate/pull/269)